src/Query.tsx

```diff
@@ -191,18 +188,8 @@ class Query extends React.Component<QueryProps, QueryState> {
       return error(result.error);
     }
 
-    if (render) {
-      return render(result);
-    }
-
-    if (typeof children === 'function') {
-      return children(result);
-    }
-
-    if (children && !isEmptyChildren(children))
-      return React.Children.only(children);
-
-    return null;
+    const renderedChildren = children(result);
+    return renderedChildren && React.Children.only(renderedChildren);
```

Comment:
Do we still have to have this limitation with React 16? We could allow more than one resulting child since now React allows it

Reply:
I'm fine with limiting to 16+ - but I'm on the progressive side.  Not sure how other users would feel about it.

Reply:
I'd say yes, since this will be part of a new version, a new API. And even then, you can _still_ use it in previous versions, only that it will fail with a less clear error when you attempt to render multiple children than the one `React.Children.only` gives you